### PR TITLE
IndexColumn: add support INDEX_LARGE

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

**Comment by komainu8** [comment]
@kenhys Could you review this PR, please?

**Comment by komainu8** [comment]
@kou Could you review this PR, please?

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed it.

File: test/test-index-column.rb [hunk 1]
```diff
@@ -452,11 +452,35 @@ def test_medium
...
+                     ])
+      end
+
       def test_invalid
         Groonga::Schema.create_table("Tags",
                                      :type => :patricia_trie,
                                      :key_type => "ShortText")
-        message = ":size must be nil, :small or :medium: <invalid>"
+        message =
+          ":size must be nil, :small, :medium, or :large: <invalid>"
```
Comment (kou) [inline test/test-index-column.rb]: Could you remove "," before "or"?

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed it.

**Pull request merged**